R/gips_perm_class.R

```diff
@@ -23,9 +23,13 @@ gips_perm <- function(x, size){
     if(!inherits(x, 'permutation'))
         x <- permutations::permutation(x)
     if(rlang::is_missing(size))
-        rlang::abort('`size` argument must be provided.')
+        rlang::abort(c("There was a problem identified with provided argument:",
+                       "i" = '`size` argument must be provided.',
+                       "x" = "You did not provide the `size` argument."))
```

Comment:
Wow, I did not know, that abort had such functionality. That's a pretty verbose message

Reply:
[More interesting functionalities](https://rlang.r-lib.org/reference/topic-condition-customisation.html)